cmake/app/boilerplate.cmake

```diff
@@ -395,8 +395,6 @@ set_property(TARGET ${IMAGE}app PROPERTY ARCHIVE_OUTPUT_DIRECTORY ${IMAGE}app)
 
 add_subdirectory(${ZEPHYR_BASE} ${__build_dir})
 
-#TODO: Everything below here.
```

Comment:
wow, wow, wow, so now everything is done? :smile:

Reply:
hehe